Maskbook/Scene/App/RedPacket/View/LuckyDropSuccessfullyView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  LuckyDropSuccessfullyView.swift
+//  Maskbook
+//
+//  Created by Hugo L on 2022/4/19.
+//  Copyright © 2022 dimension. All rights reserved.
+//
+
+import SwiftUI
+
+struct LuckyDropSuccessfullyView: View {
+    @StateObject private var viewModel = LuckyDropSuccessfullyViewModel()
+    
+    var body: some View {
+        VStack(alignment: .center, spacing: 0) {
```

Comment:
Recommend using a `VStack` like:
```
VStack(alignment: .center, spacing: 20) {
    Asset.Plugins.LuckyDrop.success.asImage()
        .resizable()
        .aspectRatio(contentMode: .fit)
        .frame(width: 60)

    VStack(spacing: 12) {
        Text("Luck Drop Successfully Created! ")
            .font(FontStyles.bh4.font)
            .foregroundColor(Asset.Colors.Text.dark.asColor())
                    
        Text("You can now see your Lucky Drop in History Tab. To let people claim it, share to your social media.")
            .font(FontStyles.rh5.font)
            .foregroundColor(Asset.Colors.Text.normal.asColor())
    }

    PrimaryButton(title: "Share to social media") {
        
    }
}
```

Reply:
It's hard to find a pattern here. 
```
|--0--|--20--|--12--|--20--|
```

Reply:
Is it a pattern of wrapping a VStack(spacing: 0) to the rest?

Reply:
This is only a recommendation. But we should leave a single line between each `View` to make code more readable.